kodi_addon_checker/xml_schema/matrix_binary_audiodecoder.xsd

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE schema PUBLIC "-//W3C//DTD XMLSCHEMA 200102//EN" "http://www.w3.org/2001/XMLSchema.dtd">
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+  <xs:element name="extension">
+    <xs:complexType>
+      <xs:attribute name="point" type="xs:string" use="required"/>
```

Comment:
Doesn't this have a fixed value for this type of add-on? Same applies to all the other new schemas

Reply:
Seems no, there comes without:
```
failed validating {'point': 'xbmc.player.musicviz', 'library_linux': 'visualization.starburst.so.2.2.0'} with XsdAttributeGroup(['library_android', 'library_freebsd', 'library_rbpi', 'library_linux', 'library_windx', 'library_windows', 'library_windowsstore', 'library_darwin_embedded', 'library_osx', 'library']):

Reason: 'point' attribute not allowed for element.

Schema:

  <xs:complexType xmlns:xs="http://www.w3.org/2001/XMLSchema">

    <xs:attribute name="library_android" type="xs:string" />
    <xs:attribute name="library_freebsd" type="xs:string" />
    <xs:attribute name="library_rbpi" type="xs:string" />
    <xs:attribute name="library_linux" type="xs:string" />
    <xs:attribute name="library_windx" type="xs:string" />
    <xs:attribute name="library_windows" type="xs:string" />
    <xs:attribute name="library_windowsstore" type="xs:string" />
    <xs:attribute name="library_darwin_embedded" type="xs:string" />
    <xs:attribute name="library_osx" type="xs:string" />
    <xs:attribute name="library" type="xs:string" />
  </xs:complexType>

Instance:

  <extension point="xbmc.player.musicviz" library_linux="visualization.starburst.so.2.2.0" />

Path: /extension
```

Reply:
Or you mean to check about name, seen it nowhere on all files and think becomes done by this https://github.com/xbmc/addon-check/blob/master/kodi_addon_checker/schema_validation.py#L50, as without correct name it comes never to the file :thinking:

Reply:
Ah you're right, we validate this one in the code and use it to match to the schema